lib/node/column-namer.js

```diff
@@ -9,6 +9,11 @@ module.exports = class ColumnNamer {
     }
 
     uniqueName(baseName) {
+        const quotedName = baseName.length > 2 && baseName[0] === '"' && baseName[baseName.length - 1] === '"';
```

Comment:
It can happen. We are now adding the `"` at the start and end of column names, so it's not possible. Previously this would have failed since you need the `"` to reference columns with spaces.

Also, using `.indexOf()` and `.lastIndexOf()` wouldn't work since `My"name"` is a valid column identifier.

Reply:
can -> can't